models/engine/db_storage.py

```diff
@@ -71,6 +71,29 @@ def reload(self):
         Session = scoped_session(sess_factory)
         self.__session = Session
 
+
+    def get(self, cls, id):
+        """Retrieve one object"""
+        objs = self.__session.query(classes[cls.__name__]).all()
```

Comment:
cls is already an object, so passing a string to an object again may be unnecessary